test/components/matrix/MatrixUsageExample2.java

```diff
@@ -0,0 +1,24 @@
+package components.matrix;
+
+public class MatrixUsageExample2 {
```

Comment:
These are slightly different. I realize one does addition and one does cross product. I'm wondering if there is a slightly more interesting use case like pixels values or something like that.